x-pack/plugins/timelines/server/search_strategy/timeline/utils/constants.ts

```diff
@@ -0,0 +1,274 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ALERT_RULE_CONSUMER } from '@kbn/rule-data-utils';
+// import { CTI_ROW_RENDERER_FIELDS } from '../../../../../../common/cti/constants';
+
+// TODO: share with security_solution/common/cti/constants.ts
+export const ENRICHMENT_DESTINATION_PATH = 'threat.enrichments';
+
+export const MATCHED_ATOMIC = 'matched.atomic';
```

Comment:
with the addition of this `constants` file, grepping for

```
export const MATCHED_ATOMIC = 'matched.atomic';
```

indicates that this would be the third copy of this constant, per the search results below:

<img width="663" alt="MATCHED_ATOMIC_search_results" src="https://user-images.githubusercontent.com/4459398/145322109-cdba41be-fd61-4068-af3b-cf3d304d956f.png">

Applying the hubristic of "the third copy is one too many", and given that (perhaps more importantly), some of the arrays in this file are _sets_ of fields that could become out of sync if a field is added to one set, but not the the other, consider reducing the number of copies of the constants in this file if that sounds reasonable without drastically increasing the scope of this PR

Reply:
Yep, forgot to delete the `events/all` file. I've removed that one in favor of the constants file now in the helpers directory. Danke!